### DISCUSSION TITLE
Data use-cases

### DISCUSSION THREAD

**Discussion by jonmjoyce** [ioos/nextgen-dmac#4] [General]
Let's discuss how data is used today. Please share some common use-cases of oceanographic data and detail the workflow. The more specific the better. Some questions to consider:

- Who is requesting the data or data product?
- What data is used? What specific fields?
- Where does the data come from? (Provide a URL if applicable)
- What processing needs to occur? Are there scripts that are run against the raw data?
- How often does this need to occur?
- Why does the requestor need this data? What purpose does this workflow fulfill?
- What are the biggest hurdles with this workflow today?
- What improvements or changes might help alleviate these hurdles?

**Comment by dpsnowden** [comment]
@jonmjoyce I think this is a critical discussion that went nowhere.  Can we discuss ways to reinvigorate this at the next steering group meeting?  I just read through the documentation and the Discussions, and we're missing a clear description of the users or user personas we're targeting, and the Data Use Cases could help us zero in on that.  This is valuable as a broader DMAC discussion, so if it doesn't fit in the timeline remaining for this project, I'd like to talk about ways to solicit broad input. We may need a github.com/ioos/use-cases repo for this type of thing. This is important inside NOAA to make the case for new resources.

**Comment by MathewBiddle** [comment]
Thanks for bringing this back up @dpsnowden.

The IOOS CodeLab can be a good starting point. https://ioos.github.io/ioos_code_lab/content/intro.html

The collection of notebooks shared there has been driven by the community and can give some insight into what types of data and use-cases folks are looking for.  It could also be a place to put the use-cases, instead of another repo.

I do have one example that I've been trying to build for the CodeLab. A notebook using EDS and sensormap to plan a surf trip, but that has proven to be a more difficult task than expected: 
* PR: https://github.com/ioos/ioos_code_lab/pull/116, 
* Here is the planned workflow: https://github.com/ioos/ioos_code_lab/issues/99#issuecomment-1185613144
